### DISCUSSION TITLE
无法变更当前渲染字体

### DISCUSSION THREAD

**Discussion by AnaheimEX** [F9y4ng/GreasyFork-Scripts#131] [问答专区(Question / Answer)]
一直是霞鹜文楷，初装脚本后选的字体，后来想换成更纱黑体 SC，但是每次在字体渲染设置里修改了字体，保存后发现还是原本的霞鹜文楷，打开字体渲染设置，也并为变更字体。

**Comment by F9y4ng** [comment]
先看看是不是同样因为这个原因造成的 #130 

**网站数据** 渲染优先级要比 **全局数据** 优先级高。

## 建议
使用前最好能阅读一下[使用说明WIKI](https://github.com/F9y4ng/GreasyFork-Scripts/wiki/%E5%AD%97%E4%BD%93%E6%B8%B2%E6%9F%93%EF%BC%88%E8%87%AA%E7%94%A8%E8%84%9A%E6%9C%AC%EF%BC%89#%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E-v500-new)

**Reply by AnaheimEX** [reply]
> 先看看是不是同样因为这个原因造成的 #130
> 
> **网站数据** 渲染优先级要比 **全局数据** 优先级高。
> 
> ## 建议
> 使用前最好能阅读一下[使用说明WIKI](https://github.com/F9y4ng/GreasyFork-Scripts/wiki/%E5%AD%97%E4%BD%93%E6%B8%B2%E6%9F%93%EF%BC%88%E8%87%AA%E7%94%A8%E8%84%9A%E6%9C%AC%EF%BC%89#%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E-v500-new)

感谢作者回复，最开始我也以为是单独网站设置的问题。
之前我访问了很多页面，发现字体都没成功变更
随后也浏览了WIKI，尝试的操作还包括删除脚本重装和重置脚本设置，都没解决。

结果之后又莫名其妙的正常了，另外直接卸载脚本好像无法彻底删除一些使用的缓冲数据？

**Reply by F9y4ng** [reply]
你用的脚本管理器是Tampermonkey吧？如果是的话，请参照 #117 第5条 回收站的设置处理。新版TM在回收站缓存处理上有些问题。

**Reply by AnaheimEX** [reply]
> 你用的脚本管理器是Tampermonkey吧？如果是的话，请参照 #117 第5条 回收站的设置处理。新版TM在回收站缓存处理上有些问题。

脚本管理器是暴力猴。。。
